joomla/joomla-cms
# [4.0] Fix undefined variable error on 'Table Options'

Pull Request for Issue # .

### Summary of Changes
Fix inine undefined variable on table options. Some minor CSS tweaks.


### Testing Instructions
Apply patch and ensure no 'Undefined variable' inline on .js-stools div in list view (Table Options).


### Before
`<div class="js-stools clearfix <br /> <b>Notice</b>: Undefined variable: dataShowOn in <b>C:\xampp\htdocs\joomla-4-dev\layouts\joomla\searchtools\default.php</b> on line <b>82</b><br /> ">`

### After
`<div class="js-stools clearfix">`


### Documentation Changes Required
